### ISSUE TITLE
Crash in production build with Vite

### ISSUE THREAD

**Issue by mitchell-vu** [EvHaus/react-pdf-charts#48]
The app was running pretty good at local machine until it was built and previewed. My project is bulid with Vite. The console throw out this error:

`Uncaught TypeError: z0.jsxDEV is not a function`

**Comment by EvHaus** [comment]
Seems to be working fine for me in Vite: https://stackblitz.com/edit/vitejs-vite-zcme9k?file=src%2FApp.tsx

Are you able to create a way for me to reproduce the problem?

**Comment by mykytahizatulin** [comment]
@EvHaus thanks for the great package, but I also got an error `Uncaught TypeError: z0.jsxDEV is not a function` only in a build mode. It's good if you run it locally without building. 
My dependencies:
 "dependencies": {
    "@react-pdf/renderer": "^3.1.12",
    "@reduxjs/toolkit": "^1.9.2",
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
    "react": "^18.2.0",
    "react-beautiful-dnd": "^13.1.1",
    "react-dom": "^18.2.0",
    "react-pdf-charts": "^0.0.5",
    "react-scripts": "5.0.1",
    "recharts": "^2.7.2",
  },

so basically when you run `npm run react-scripts build` and then serve the app you'll get this error, but if you run `npm run react-scripts start` it works great. I'd appreciate any help with that :)

**Comment by EvHaus** [comment]
Thanks! I think I found the issue. I just published v0.2.1 that should hopefully fix the problem. Let me know if it's still happening.
